deepmd/entrypoints/main.py

```diff
@@ -271,6 +271,13 @@ def main_parser() -> argparse.ArgumentParser:
         type=str,
         help="The system dir. Recursively detect systems in this directory",
     )
+    parser_tst.add_argument(
+        "-f",
```

Comment:
It seems that -f and -s can not exist at the same time. 
It's better to use 
'''
parser_tst_subgroup = parser_tst.add_mutually_exclusive_group()
parser_tst_subgroup.add_argument("-s", ... 
parser_tst_subgroup.add_argument("-f", ...
'''